Dependencies/ShaderFixes/uscp.hlsl

```diff
@@ -0,0 +1,9 @@
+Texture2D<float4> t101 : register(t101);
```

Comment:
I'm probably missing something, but can we just add this code directly to the sbs PS instead?

I'd rather change the signature of the HLSL PS and VS than add two more files.  

It's OK if the output of the VS isn't always used.